action.yml

```diff
@@ -11,6 +11,12 @@ inputs:
   finalize:
     description: 'When false, omit marking the release as finalized and released.'
     default: true
+  ignore_missing:
+    description: 'When the flag is set and the previous release commit was not found in the repository, will create a release with the default commits count instead of failing the command.'
+    required: false
```

Comment:
`ignore_missing` and `ignore_empty` default to `false`. Do we have to update the `action.yml` to add a `default: true` for these fields?

Reply:
We don't, as we do that here https://github.com/getsentry/action-release/pull/63/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR15-R16